ccpp/CCPP_driver.F90

```diff
@@ -56,14 +56,26 @@ subroutine CCPP_step (step, nblks, ierr)
     character(len=*),         intent(in)  :: step
     integer,                  intent(in)  :: nblks
     integer,                  intent(out) :: ierr
+    character(len=*),         intent(in),optional  :: dynamics
```

Comment:
I would prefer this to be a mandatory argument, not an optional one.

Reply:
Agreed. Will change.

Reply:
Done.